packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -225,12 +226,8 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
             {errorMessage && (
               <div aria-live="assertive">
                 <DelayedRender>
-                  <p
-                    className={css('ms-TextField-errorMessage', AnimationClassNames.slideDownIn20, styles.errorMessage)}
-                  >
-                    <span className={styles.errorText} data-automation-id="error-message">
```

Comment:
I'm not sure if you're saying it can be removed or if you think it was removed. It's still in `ITextFieldStyles` with default global class name in `TextField.styles.ts'.

Reply:
I am asking if it should be missing or not.  I am talking about the **span** tag style `styles.errorText`, not the **p** tag style `styles.errorMessage` which is reapplied on 229

Reply:
Actually, I'm not seeing it in the original scss do I must be missing something

Reply:
Ahh I see. Yeah, I didn't see `errorText` being defined anywhere. The fact that there wasn't an error here is a problem of using the old `any` styles. :/

Reply:
Oh is that why there is no error?! Good to know I was racking my brain trying to figure out how that slipped by.  In that case, must just be left over from some point when it did have styles.